jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/websocket/WebSocketConstants.java

```diff
@@ -43,19 +43,25 @@
 
     public static final String DIAGNOSTIC_PATH_PARAMS_ANNOT_MISSING = "Parameters of type String, any Java primitive type, or boxed version thereof must be annotated with @PathParams.";
     public static final String DIAGNOSTIC_CODE_PATH_PARMS_ANNOT = "AddPathParamsAnnotation";
-    
+
     public static final String DIAGNOSTIC_ON_OPEN_INVALID_PARAMS = "Invalid parameter type. Parameter must be of type: \n- jakarta.websocket.Session \n- jakarta.websocket.EndpointConfig \n- annotated with @PathParams and of type String or any Java primitive type or boxed version thereof";
     public static final String DIAGNOSTIC_CODE_ON_OPEN_INVALID_PARAMS = "ChangeInvalidParam";
-    
+
+    public static final String DIAGNOSTIC_SERVER_ENDPOINT_NO_SLASH = "Server endpoint paths must start with a leading '/'.";
+    public static final String DIAGNOSTIC_SERVER_ENDPOINT_NOT_LEVEL1 = "Server endpoint paths must be a URI-template (level-1) or a partial URI.";
+    public static final String DIAGNOSTIC_SERVER_ENDPOINT_RELATIVE = "Server endpoint paths must not contain the sequences /../, /./ or //.";
```

Comment:
for consistency and clarity, it would be nice to place the actual sequences within single quotes like the diagnostic message on line 50. Specifically, the last one might be confusing and a user might think the sequence is '//.' rather than just '//'

Reply:
oh that's a good point